core/src/main/java/org/apache/iceberg/rest/CatalogHandlers.java

```diff
@@ -336,4 +348,8 @@ private static TableMetadata commit(TableOperations ops, UpdateTableRequest requ
 
     return ops.current();
   }
+
+  private static String metadataLocation(Table tbl) {
+    return ((HasTableOperations) tbl).operations().current().metadataFileLocation();
+  }
```

Comment:
Currently unused - was using to check the metadata file location in some of the failing tests. It's only intermediate results that have a `LoadTableResponse` without a metadataLocation (e.g. if `stageCreate` is being called).

So maybe `metadata-location` should be nullable as `LoadTableResponse` can be returned without it? Or we need to be adding it somewhere in the responses that we're not inside of `RESTSessionCatalog`.